po/hi.po

```diff
@@ -0,0 +1,864 @@
+# Hindi translation for avahi.
+# Copyright (C) 2024 avahi's COPYRIGHT HOLDER
+# This file is distributed under the same license as the avahi package.
+# Scrambled 777 <weblate.scrambled777@simplelogin.com>, 2024.
+# Scrambled777 <weblate.scrambled777@simplelogin.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: avahi master\n"
+"Report-Msgid-Bugs-To: https://github.com/lathiat/avahi/issues\n"
```

Comment:
Could you change it to https://github.com/avahi/avahi/issues? avahi moved to the avahi organization recently.

Reply:
Sure. Will update `po` file now. Thanks.